score/RunScore.py

```diff
@@ -0,0 +1,48 @@
+from score.ScoreBonus import *
+from save import Savefile
+
+
+class RunScore:
```

Comment:
Any reason this isn't a method on `FileParser`? The constructor looks like it's mostly unneeded, and instead should be a dict of `role_name: ClassName`, which we can grab for each score bonus.